x/market/internal/types/pool_info.go

```diff
@@ -0,0 +1,27 @@
+package types
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// PoolInfo defines information to describe each denomination pool
+type PoolInfo struct {
+	Delta            sdk.Dec `json:"delta" yaml:"delta"`
+	RegressionAmount sdk.Dec `json:"regression_amount" yaml:"regression_amount"`
+}
+
+func NewPoolInfo(delta sdk.Dec, regressionAmount sdk.Dec) PoolInfo {
+	return PoolInfo{
+		Delta:            delta,
+		RegressionAmount: regressionAmount,
+	}
+}
+
```

Comment:
Add // Extends fmt.Stringer comment to help the linter

Reply:
Done